testcases/commands/logrotate/logrotate_tests.sh

```diff
@@ -40,342 +40,148 @@
 #                Dec 24 2002 - Added   - Test #2 - Test to run logrotate as a
 #                                        cron job.
 #                Feb 28 2003 - Fixed   - Modified testcase to use functions.
+#                Mar 04 2024 - Fixed   - Rewrite according to the new guideline.
 #
-# Function: 	chk_ifexists
-#
-# Description:  - Check if command required for this test exits.
-#
-# Input:        - $1 - calling test case.
-#               - $2 - command that needs to be checked.
-#
-# Return:		- zero on success.
-# 				- non-zero on failure.
-chk_ifexists()
-{
-	RC=0
-
-	which $2 > $LTPTMP/tst_logrotate.err 2>&1 || RC=$?
-	if [ $RC -ne 0 ]
-	then
-		tst_brkm TBROK NULL "$1: command $2 not found."
-	fi
-	return $RC
-}
 
+TST_NEEDS_CMDS="logrotate egrep file"
```

Comment:
I guess just `grep` (and use  `grep -E` instead of `egrep`. Also please sort alphabetically.